cluster/ci/config/prow/jobs/cluster-launch-e2e.yaml

```diff
@@ -183,6 +184,12 @@ objects:
           -e openshift_test_repo=${RPM_REPO} -e 'oreg_url=${IMAGE_FORMAT}' \
           playbooks/${CLUSTER_TYPE}/openshift-cluster/launch.yml
 
+        if [[ -n "${ANSIBLE_RUN_UPDATE-}" ]]; then
```

Comment:
I don't think this is appropriate for here.  We already have an upgrade playbook in the repo.

Reply:
Okay, I'll create templates for minor and major upgrades testing